### PR TITLE
Merge TableTop into v.next

### PR DESCRIPTION
**Pull request by gunt0001**
Related to issue: https://devtopia.esri.com/runtime/kotlin/issues/4900

### Description:

Merge `ar` module into `v.next`, which includes:
- `TableTopSceneView` and related classes
- TableTop micro app

### Pre-merge Checklist

- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/) Job for this PR has been run
  - [x] link: https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/197/
- Unit and/or integration tests have been added to exercise this PR's logic, and the tests are passing:
  - [ ] Yes
  - [ ] No

### CODE REVIEW

**Review by puneet-pdx — APPROVED** [review]
@gunt0001 LGTM 👍🏽

**Review by alan-edi — APPROVED** [review]
@gunt0001 Just a few detailed comments to consider. I'll approve it so merging isn't held up by my absence tomorrow.

File: toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/render/Texture.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+import android.graphics.BitmapFactory;
+import android.opengl.GLES11Ext;
+import android.opengl.GLES30;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+/**
+ * A GPU-side texture. This class is based on the [Texture](https://github.com/google-ar/arcore-android-sdk/blob/main/samples/hello_ar_kotlin/app/src/main/java/com/google/ar/core/examples/kotlin/helloar/HelloArRenderer.kt)
```
[the remark this answers is not in the document]
Comment (gunt0001) [reply toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/render/Texture.java]: updated

File: toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/render/VertexBuffer.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+ */
+package com.arcgismaps.toolkit.ar.internal.render;
+
+import android.opengl.GLES30;
+
+import java.io.Closeable;
+import java.nio.FloatBuffer;
+
+/**
+ * A list of vertex attribute data stored GPU-side. This class is based on the [VertexBuffer](https://github.com/google-ar/arcore-android-sdk/blob/main/samples/hello_ar_kotlin/app/src/main/java/com/google/ar/core/examples/kotlin/helloar/HelloArRenderer.kt)
```
[the remark this answers is not in the document]
Comment (gunt0001) [reply toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/render/VertexBuffer.java]: updated

**Pull request merged**